src/embind/embind.js

```diff
@@ -2186,14 +2187,14 @@ var LibraryEmbind = {
         var humanName = classType.name + '.' + fieldName;
         var desc = {
             get: function() {
-                throwUnboundTypeError('Cannot access ' + humanName + ' due to unbound types', [getterReturnType, setterArgumentType]);
+                throwUnboundTypeError('Cannot access ' + humanName + ' due to unbound types', [rawFieldType]);
```

Comment:
this looks like an actual change, not just fixing a closure warning? i'm not familiar enough with the embind code to know if it's valid or not, but was this intentional?

Reply:
Closure Compiler complained about undefined variables and I've made a change that seems appropriate (I'm not familiar with this part of the code base either). Looks like a copy-paste from few pages of code above before this place.
The lines were introduced in 1f6b09dc8e50aba1222887f88a621b67ba86e168 by @chadaustin, could you confirm this is a correct fix? Anyway, this is not a main code path, so doesn't seem that critical.

Reply:
also cc @AlexPerrot 

It might make sense to split this off into another embind-focused PR (but let's see what they say first, maybe not)

Reply:
This change looks fine to me.  I'm a little bit surprised that this made it in there in the first place (since we had linters at the time) but who knows.  :)